charts/lighthouse/templates/tekton-controller-role.yaml

```diff
@@ -5,50 +5,33 @@ metadata:
   name: {{ template "tektoncontroller.name" . }}
 rules:
 - apiGroups:
-  - ""
+  - tekton.dev
   resources:
-  - namespaces
-  - configmaps
-  - secrets
+  - pipelines
   verbs:
   - get
-  - list
```

Comment:
Yup, need this back.

Reply:
Hmmm yeah... controller-runtime cache again, I need a non caching client instead.

Reply:
Should be fixed 🤞